src/main/java/org/fundacionjala/trello/driver/RemoteDriver.java

```diff
@@ -0,0 +1,36 @@
+package org.fundacionjala.trello.driver;
+
+import org.fundacionjala.core.RemoteServerEnv;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+
+public final class RemoteDriver extends AbstractBrowser {
```

Comment:
How can a "driver" extend from a "browser"? 
Maybe rename this to just "Remote" as we did for "Chrome" and "Headless"?